lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -253,6 +258,28 @@ sub _route_path {
   return $path;
 }
 
+sub _security_action {
+  my ($self, $security_settings) = @_;
+  my $security_cb = $self->_security_cb;
+
+  return sub {
+    my $c = shift;
+    my @cc
+      = map { my ($name, $config) = %$_; ($security_cb->{$name}, $config); } @$security_settings;
+    my $wrapper;
+
+    $wrapper = sub {
+      my $c = shift;
+      my $cb = shift @cc || sub { die 'No security handler defined' };
+      $c->$cb(shift @cc, $wrapper);
```

Comment:
I was also considering doing:

```
# App wide, instead of plugin wide:
$c->openapi->security->dummy($config, $wrapper);
# Could use something like "openapi_security_dummy" by default, but then be overridden by "x-mojo-helper" or something.
$c->$helper_name($config, $wrapper);
```